sympy/solvers/ode/tests/test_ode.py

```diff
@@ -33,6 +33,152 @@
 # dsolve() to return different results on different machines)
 
 
+def _ode_solver_test(ode_examples,our_hint='default'):
```

Comment:
I guess that this function doesn't need to take the hint as an argument any more

Reply:
okay, updating it now.